apisix/plugins/mocking.lua

```diff
@@ -229,6 +229,7 @@ function _M.access(conf, ctx)
 
     if conf.response_headers then
         for key, value in pairs(conf.response_headers) do
+            value = core.utils.resolve_var(value, ctx.var)
```

Comment:
`resolve_var` may return nil. Please cover this case. Also add test case for this situtation,